app/docker/components/log-viewer/logViewerController.js

```diff
@@ -7,6 +7,7 @@ function (clipboard) {
     copySupported: clipboard.supported,
     logCollection: true,
     autoScroll: true,
+    displayTimestamp: false,
```

Comment:
No need to store the value in this controller, we're going to use the binded value directly. You can remove it.